src/shared/netioapi.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright © 2015-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::basetsd::SIZE_T;
+use shared::guiddef::GUID;
+use shared::ifdef::{NET_IFINDEX,NET_LUID,PNET_IFINDEX,PNET_LUID};
```

Comment:
There should be a space after the comma.

Reply:
Fixed